extensions/util/pool.go

```diff
@@ -23,69 +23,68 @@ import (
 	"github.com/xo/dburl"
 )
 
-var GlobalPool *driverPool
+var GlobalPool *dbPool
 
 func init() {
 	// GlobalPool maintained the *sql.DB group by the driver and DSN.
 	// Multiple sql sources/sinks can directly fetch the `*sql.DB` from the GlobalPool and return it back when they don't need it anymore.
 	// As multiple sql sources/sinks share the same `*sql.DB`, we can directly control the total count of connections by using `SetMaxOpenConns`
-	GlobalPool = newDriverPool()
+	GlobalPool = newDBPool()
 }
 
-type driverPool struct {
+type dbPool struct {
 	isTesting bool
 
 	sync.RWMutex
-	pool map[string]*dbPool
+	// url -> *sql.DB
```

Comment:
The db url could have properties like: protocol//[hosts][/database][?properties]. So using url as key, two url with the same protocol, hosts, database may have different db instance. Is this expected?

Reply:
Yes, I think we can keep this behavior for now.